components/x11/x11-protocols/x11-protocols.p5m

```diff
@@ -0,0 +1,327 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2019 Aurelien Larcher
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.description value="Developer documentation and C language definitions of X11, X extension, and related protocols"
+set name=pkg.summary value="$(COMPONENT_SUMMARY)"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+# Provided by xorgproto
+#depend fmri=x11/header/bigreqsproto type=require
```

Comment:
As I understand, a dozen *proto* was merged into x11-protocols. 
To do this update correctly you need to add to history file (in x11-protocol component or meta-packages/history, doesn't matter much), a pair old_FMRI@(old_version+1) NEW_FMRI.
This will lead to generation of dozen rename packages.

To be pedantic you could make new package depend on this old_FMRI@(old_version+1) package, so that installation of new package lead to deinstallation of old packages.

Reply:
OK so my mistake is that I did not bump the version in the history file...

Reply:
Your mistake is package names - x11/header/xf86bigfontproto. 
Also you've sorted history file, this makes reviewing harder.